src/components/Banner.tsx

```diff
@@ -1,8 +1,12 @@
+import { EntityFieldType, resolveYextEntityField, YextEntityFieldSelector } from "@yext/visual-editor";
```

Comment:
Can you also update location.tsx to remove the type from config and put `as const satisfies TemplateConfig` at the end?